frontend/class-breadcrumbs.php

```diff
@@ -895,33 +895,4 @@ private function get_output_class() {
 
 		return $class;
 	}
-
-	/********************** DEPRECATED METHODS **********************/
```

Comment:
I'm not sure if we want to remove this header as we might have deprecated methods in the feature.